src/fleet/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.2.7'
+VERSION = '0.2.9'
```

Comment:
Please follow this [doc](https://microsoft.sharepoint.com/:w:/t/AzureCLITools237/EWG5EuACKmtNum6A3pyLakMBSaQnkjomC9XPz9ogNPS69A?e=zRopkS) for the versioning of extension.

Reply:
Updated to 0.3.0